test/Interop/Cxx/class/memory-layout-silgen.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend -I %S/Inputs -enable-cxx-interop -emit-ir -o - %s | %FileCheck %s
+// RUN: %target-swift-frontend -I %S/Inputs -enable-cxx-interop -Xcc -nostdinc++ -emit-ir -o - %s | %FileCheck %s
```

Comment:
I found this workaround for the C++ header issues, which works both when cross-compiling using the Android NDK and when natively running this test from the Swift test suite in the Termux app on Android.

The issue was that the previous Android NDK layout didn't include the C++ headers in the C sysroot but in a completely separate directory `$NDK/sources/cxx-stl/llvm-libc++/include/`, which wasn't included in these Android cross-compilation tests. Now that we're switching to the new NDK unified sysroot, all C and C++ headers are included in the same sysroot, so clang started including the C++ headers for this test and breaking.

This flag disables including the C++ headers, at least until that starts working. @zoecarver or @martinboehme, could you confirm that this is okay?